kite-data/kite-data-crunch/pom.xml

```diff
@@ -318,5 +318,36 @@
         </plugins>
       </build>
     </profile>
+    <profile>
+      <id>hdp2-release</id>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-deploy-plugin</artifactId>
```

Comment:
I'm still confused why we need these. I think it has something to do with Kite shading Guava, as described here: https://github.com/scalingdata/cdk/pull/20#discussion_r40102178

Though I'm sure you'll explain it very well I think this config is unusual enough that others may get confused too. Maybe worth a comment?